### DISCUSSION TITLE
when using --wsl: Change the conda env to one created in wsl.

### DISCUSSION THREAD

**Discussion by meicale** [neovide/neovide#2268] [General]
**Is your feature request related to a problem? Please describe.**
I can use plugins such as vim-conda to change conda environment in neovim running on terminal. BUT, I cannot use this plugins when using neovide GUI running on Windows with neovim instance running in WSL2.(by using --wsl) This plugins just failed. If this ability is supported by Neovide, this will be much better.
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by fredizzimo** [comment]
This is a configuration issue, so changing to a discussion
